.travis.yml

```diff
@@ -1,7 +1,9 @@
 language: python
 python:
     - "2.7"
-env: PGVERSION=9.1
+env:
+    - PGVERSION=9.1 CKANVERSION=master
```

Comment:
Minor improvement:
I would name this env variable 'CKAN_BRANCH' with the values `master` and `release-v2.5.1` and later on use it as-is in the `git checkout` command (and avoid the if).